sockeye/vocab.py

```diff
@@ -32,23 +32,42 @@
 InverseVocab = Dict[int, str]
 
 
-def build_from_paths(paths: List[str], num_words: Optional[int] = None, min_count: int = 1,
-                     pad_to_multiple_of: Optional[int] = None) -> Vocab:
+def build_from_paths(paths: List[str]) -> Counter:
```

Comment:
It seems it can either be a single value or a list, so I think we should use the `Union` type from `typing` which specifies this:
```suggestion
def build_from_paths(paths: Union[List[str], str]) -> Counter:
```

Reply:
This should also then bubbled up if it applies to callers.